archunit/src/main/java/com/tngtech/archunit/core/domain/JavaMethod.java

```diff
@@ -15,20 +15,20 @@
  */
 package com.tngtech.archunit.core.domain;
 
-import java.lang.reflect.Method;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
-import java.util.function.Function;
-import java.util.function.Supplier;
-
 import com.tngtech.archunit.PublicAPI;
 import com.tngtech.archunit.base.ArchUnitException.InconsistentClassPathException;
 import com.tngtech.archunit.base.MayResolveTypesViaReflection;
 import com.tngtech.archunit.base.ResolvesTypesViaReflection;
 import com.tngtech.archunit.base.Suppliers;
 import com.tngtech.archunit.core.importer.DomainBuilders;
 
+import java.lang.reflect.Method;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
```

Comment:
Can we please adhere to the import order specified in [`CONTRIBUTING.md`](https://github.com/TNG/ArchUnit/blob/main/CONTRIBUTING.md), i.e. keep the `java.*` imports at the top?

Reply:
Cool, I'll do so in `JavaMethodTest` as well